index.ts

```diff
@@ -0,0 +1,81 @@
+import { NativeModules } from 'react-native'
+
+const { RNZoomUs } = NativeModules
+
+if (!RNZoomUs) console.error('RNZoomUs native module is not linked.')
```

Comment:
```suggestion
if (!RNZoomUs) throw new Error('RNZoomUs native module is not linked.')
```

Reply:
No, no, no. Very bad idea.

Reply:
why is that?. Better to throw early in dev mode than when using join/start/init function, no?

Reply:
In dev mode, console.error will show annoying modal. It is good enough and we can continue work even without rebuilding app.

If something will happen in release build, app still can work.

I don't want to crash app when someone has issues with auto/manual linking. Since they understand where is issue.